src/models/blueprint-collection.js

```diff
@@ -0,0 +1,100 @@
+import path from 'path';
+import walkSync from 'walk-sync';
+import fs from 'fs';
+import _map from 'lodash/map';
+import _filter from 'lodash/filter';
+import _isNil from 'lodash/isNil';
+import _isBool from 'lodash/isBoolean';
+import _isString from 'lodash/isString';
+import _isArray from 'lodash/isArray';
+import _flatten from 'lodash/flatten';
+import _uniq from 'lodash/uniq';
+
+export default class BlueprintCollection {
+  constructor (settings) {
+    this.settings = settings;
+    this.setSearchPaths();
+  }
+
```

Comment:
Seeing 'settings' here was a mental disconnect for me, my first thought was that it was the ProjectSettings object, which it isn't, only the configured search paths.

Reply:
good point.   I'll rename that to something like rawPaths or potentialPaths